Update device and plattform testing configuration

moiikana opened the pull request:
- we needed to update the test setup for the automated browser testing (testingbot), because in our configuration the browsers/devices were outdated or not supported by testingbot anymore

- this makes the setup more generic and easier to update

https://phabricator.wikimedia.org/T383402

moiikana committed:
Update device and plattform testing configuration

- we needed to update the test setup for the automated browser testing (testingbot), because in our
configuration the browsers/devices were outdated or not supported by testingbot anymore

- this makes the setup more generic and easier to update

- remove one of the desktop resolutions because there were <3% users with a width around 1000 pixels

https://phabricator.wikimedia.org/T383402

gbirke commented on campaign_info.toml:
```diff
@@ -84,7 +84,7 @@ pagename = "B25_WMDE_iPad_DE_00_var"
 
 [pad.test_matrix]
-device = [ 'ipad_mini', 'ipad', 'ipad_pro_9_7_inch', 'ipad_pro_12_inch' ]
+device = [ 'ipad_mini', 'ipad_pro' ]
```
There are 3 iPad sizes - ipad mini (8.3"), iPad (10,9") and iPad Pro/iPad Air (13"). 
See https://www.apple.com/de/ipad/compare/?modelList=ipad-pro-13-m4,ipad-10th-gen,ipad-mini-a17pro

Since one of purposes of the screenshots is discovering visual glitches at different resolutions, I'd be in favor of adding the "mid" size (named just `'ipad'`) here as well. I looked at the testingbot options - as long as we don't use iOS 18, we can use the iPad (10th Generation) as the mid-size model.

moiikana replied:
yes, thank you. I'm still looking into mobile issues and will look into ipad next.

my current problem with the ipad setup is, that testingbot is giving mixed signals on what they provide.
on the "live testing" interface it seems very limited (no mini):
![Bildschirmfoto vom 2025-01-21 13-12-42](https://github.com/user-attachments/assets/14a7945b-a276-4b39-982f-d10727ed7848)

in the documentation however they let you choose more ipads in "Specify Browsers&Devices":
https://testingbot.com/support/getting-started/php.html

moiikana replied:
found answers to my questions on https://testingbot.com/support/getting-started/browsers.html and created device configurations for 3 different ipad sizes

moiikana committed:
Remove old android test platforms

- devices with firefox are currently not responsive at testingbot's site, the old phone testing is also not needed

moiikana commented:
dropped commit with ipad/iPad/pad unification attempt
